src/trace-writer.ts

```diff
@@ -74,9 +76,10 @@ export class TraceWriter extends common.Service {
    */
   constructor(
       private readonly config: TraceWriterConfig,
-      private readonly logger: common.Logger) {
+      private readonly logger: Logger) {
     super(
         {
+          requestModule: request,
```

Comment:
@DominicKramer Why do we add this line?

Reply:
The main change in the upgrade to the newest version of `@google-cloud/common` is the introduction of the `requestModule` config.  Dependents of `@google-cloud/common` need to specify explicitly what request module they want to use.

For our other modules, we have first used `request`.  Then, in a follow-up PR, we switched to using `teeny-request` to improve module require time.